ext/curl/multi.c

```diff
@@ -251,7 +251,7 @@ PHP_FUNCTION(curl_multi_exec)
 
 	ZEND_PARSE_PARAMETERS_START(2, 2)
 		Z_PARAM_RESOURCE(z_mh)
-		Z_PARAM_ZVAL_DEREF(z_still_running)
+		Z_PARAM_ZVAL(z_still_running)
```

Comment:
This demonstrates another significant API requirement.
Each third-party extension, that uses reference parameters, has to be adopted.